.build/.gitignore

```diff
@@ -1 +0,0 @@
-sizes.json
```

Comment:
please re-remove sizes.json. We should be generating this every time we build, so this shouldn't be needed